e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -847,9 +862,9 @@ export class ExtrinsicHelper {
     const blockNumber = ofBlockNumber || (await getBlockNumber());
     let currentBlock = await getFinalizedBlockNumber();
     while (currentBlock < blockNumber) {
-      if (start + 48_000 < Date.now()) {
+      if (start + 120_000 < Date.now()) {
```

Comment:
nit: Seems like, as this is in several places, it could benefit from being refactored as a constant...

Reply:
It is just in these two I believe. I was tempted to turn it into a function that returns a different number for different environments, but that felt like too much.